pkg/security/registry/podsecuritypolicysubjectreview/rest.go

```diff
@@ -42,22 +45,29 @@ func (r *REST) Create(ctx kapi.Context, obj runtime.Object) (runtime.Object, err
 	if !ok {
 		return nil, kapierrors.NewBadRequest(fmt.Sprintf("not a PodSecurityPolicySubjectReview: %#v", obj))
 	}
-
 	ns, ok := kapi.NamespaceFrom(ctx)
 	if !ok {
 		return nil, kapierrors.NewBadRequest("namespace parameter required.")
 	}
-
 	if errs := securityvalidation.ValidatePodSecurityPolicySubjectReview(pspsr); len(errs) > 0 {
 		return nil, kapierrors.NewInvalid(kapi.Kind("PodSecurityPolicySubjectReview"), "", errs)
 	}
+	groups := pspsr.Spec.Groups
+
+	if len(groups) == 0 {
+		if actualGroups, err := r.groupCache.GroupsFor(pspsr.Spec.User); err == nil {
```

Comment:
I think we don't want to auto-infer any groups. the linked bug seems to be an incorrect use of the API